.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "eslint.format.enable": true,
+  "eslint.trace.server": "off",
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  }
+}
```

Comment:
Remove this please :)

Reply:
Hey @tristanMatthias, vscode config has been removed and also added to .getignore. I believe I've updated it appropriately. Thanks.